api-presigned.go

```diff
@@ -84,19 +84,40 @@ func (c Client) presignURL(method string, bucketName string, objectName string,
 }
 
 // PresignedGetObject - Returns a presigned URL to access an object
-// without credentials. Expires maximum is 7days - ie. 604800 and
-// minimum is 1. Additionally you can override a set of response
-// headers using the query parameters.
+// data without credentials. URL can have a maximum expiry of
+// upto 7days or a minimum of 1sec, if expiry duration is set to 0
+// URL a maximum value of 7 days is automatically choosen. Additionally
```

Comment:
+// upto 7days or a minimum of 1sec, if expiry duration is set to 0
 +// URL a maximum value of 7 days is automatically choosen.

Should be:
+// upto 7days or a minimum of 1sec, if expiry duration is set to 0
 +// ~~URL~~ a maximum value of 7 days is automatically cho~~o~~sen.

Ditto for the other 3 function header comments